templates/pages/links.html

```diff
@@ -4,10 +4,16 @@
 
 {% block content%}
 
-	<div>
-		<p>We gather useful links to tutorials, good practices, etc. </p>
-
+    <div align="center" class="naslov">
```

Comment:
Same comment as below - you defined a custom class ("naslov"), so it's not good practice to define alignment right in HTML (HTML is for writing content, CSS is for style). So, just define alignment and other visual behavior in the CSS file.

Also, I'd recommend keeping class names in English :)